Runtime/IRuntimeConfigurationExtensions.cs

```diff
@@ -1,36 +0,0 @@
-﻿using Innoactive.Creator.Core.Configuration;
-
-namespace Innoactive.Creator.TextToSpeech
-{
-    /// <summary>
-    /// TextToSpeech extensions methods for <see cref="IRuntimeConfiguration"/>.
-    /// </summary>
-    public static class IRuntimeConfigurationExtensions
```

Comment:
API break. I suggest keeping this class alongside the new one, but mark it as obsolete.

Reply:
(This is an API break because you can call these methods just like normal static methods)